code/controllers/subsystem/air.dm

```diff
@@ -361,7 +360,7 @@ SUBSYSTEM_DEF(air)
 		while (turfs_to_check.len)
 		var/ending_ats = active_turfs.len
 
-		var/msg = "HEY! LISTEN! [DisplayTimeText(world.timeofday - timer)] were wasted processing [starting_ats] turf(s) (connected to [ending_ats] other turfs) with atmos differences at round start."
```

Comment:
I'll miss this tbh

Reply:
Same, but it's currently printing HEY! LISTEN! were wasted, and I think it's because of the atmos changes. I can change it to something more flavorful, but the world timer is no longer working